queries.sql

```diff
@@ -1,3 +1,35 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals
+WHERE name LIKE '%mon'
+
+SELECT name
+FROM animals
+WHERE date_of_birth BETWEEN 2016 AND 2019;
+
+SELECT name
+FROM animals
+WHERE date_of_birth BETWEEN 2016 AND 2019;
+
+SELECT name
+FROM animals
+WHERE neutered=true && AND escape_attempts < 3;
```

Comment:
- [OPTIONAL] Why the use of `&&` and `AND` when you can simply do `AND` which would be enough.

Reply:
Thanks @mmsesay 
I just forget to remove the `&&`.